C#/cs01/01.cs

```diff
@@ -1,11 +1,46 @@
 using System;
 namespace Brainchild.Curriculum.Solutions
 {
-    public class Solution01
+    /*
+    Write a program that prints the numbers from 1 to 100 and for multiples of '3' print "Fizz" instead of the number 
+    and for the multiples of '5' print "Buzz".
+
+    1, 2, Fizz, 4, Buzz, Fizz, 7, 8, Fizz, Buzz, 11, Fizz, 13, 14, 
+    Fizz ...
+    */
+    public class FizzBuzzPrinter
     {
-        public bool ToDo(int a, float b,string c)
-        {
-            throw new NotImplementedException("Please write your solution here");
+        ///Accept one number and return a string(Fizz or Buzz or the number ) based on the aforementioned logic 
+        public string Print(int num)
+         {
+             if(num%3==0){
```

Comment:
indentation is correct but spacing needs to be corrected everywhere.
`num%3 == 0` (notice the spaces before and after ==. makes the code more readbale